### PR TITLE
Invalid extension alert and clear input

### LINKED ISSUE #15541: upload function
# Describe the bug
in import.php 
when i import new database  there is no validation while the uploaded extension , for example i uploaded img and i name it like that : <svg/onload=alert(document.cookie)> 

and i see the successful upload image and the app try to show me that error but there isn't any shown errors from it  

### To Reproduce

1. Go to 'import.php'
2. Click on 'select database'
3. Scroll down to 'select the named image'

### Expected behavior

 show me the error from inside the image , i injected evil function with php 

### Server configuration

- Operating system: win10
- phpMyAdmin version: 4.9.1

### Client configuration

- Browser: firfox
- Operating system: win

### Additional context

as you can see in the uploaded img 
on of them show the error from the content file the uploaded php file 
but when i create image with that name it's uploaded but can't read the evil function  from the image

### PR DESCRIPTION
**Pull request by tlshifat**
Signed-off-by: Md Shifaul Islam <shifat.it@gmail.com>

### Description

Invalid extension alert and clear input to upload valid files again. 

Fixes #15541

Before submitting pull request, please review the following checklist:

- [x] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [x] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [x] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [x] Every commit has a descriptive commit message.
- [x] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [x] Any new functionality is covered by tests.

### CODE REVIEW

**Comment by shifatbuet** [comment]
Please proceed . Existing flow is ok . Just an alert given for invalid extension. 
Hope this will help.

File: js/import.js [hunk 1]
```diff
@@ -30,6 +30,15 @@ function matchFile (fname) {
     var len = fnameArray.length;
     if (len !== 0) {
         var extension = fnameArray[len - 1];
+        // list of valid extension
+        var validExt = ['csv','sql','xml','txt','ods','atx','ain','aih','dbf','gz','zip','bz2'];
+        if (!validExt.includes(extension)) {
+            // alert error response
+            alert(
+                'Invalid file format . Valid file format (\'csv\',\'sql\',\'xml\',\'txt\',\'ods\',\'atx\',\'ain\',\'aih\',\'dbf\',\'gz\',\'zip\',\'bz2\')\n');
```
Comment (williamdes) [inline js/import.js]: This fix is only a front-end one, using an extension like [TamperMonkey](https://chrome.google.com/webstore/detail/tampermonkey/dhdgffkkebhmkfjojejmpbldmpobfkfo) I can easily remove this check.

So IMHO you need to do a back-end fix using php code

**Review by williamdes — CHANGES_REQUESTED** [review]
Please do a php server side check

But this check is restricting users as @ibennetch said

File: js/import.js (hunk 1 above)
Comment (shifatbuet) [reply js/import.js]: Okay.

Comment (shifatbuet) [reply js/import.js]: I am working on it. @williamdes

Comment (williamdes) [reply js/import.js]: Thank you @shifatbuet

**Review by williamdes — COMMENTED** [review]
Thank you for the server side fix !

File: js/import.js (hunk 1 above)
Comment (ibennetch) [reply js/import.js]: On the other hand, IMHO, this is really meant to protect a user from an accidental click; there's no real attack vector here that I can see. So doing this on client side would make more sense, so as to save the upload time to the server.

Comment (shifatbuet) [reply js/import.js]: Both client side and server side check is done. Please proceed.

**Comment by williamdes** [comment]
Sorry but I will not merge this PR
IMO users do what they want and if they upload bad files no problem if it does not introduce a security issue.

**Pull request closed without merging**